<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntAlgorithmDialog</class>
 <widget class="QDialog" name="AntAlgorithmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ant Colony Algorithm Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleEvaporation">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelEvaporation">
     <property name="text">
      <string>Коэффициент испарения феромона</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="comboEnd"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelBeta">
     <property name="text">
      <string>Влияние эвристики</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelAlpha">
     <property name="text">
      <string>Влияние феромона</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelStart">
     <property name="text">
      <string>Начальная точка</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelQ">
     <property name="text">
      <string>Количество феромона</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelNumAnts">
     <property name="text">
      <string>Число муравьев</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboStart"/>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="doubleQ">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinNumIterations">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="doubleBeta">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelEnd">
     <property name="text">
      <string>конечная точка</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelPercentage">
     <property name="text">
      <string>процент схождения</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="spinPercentage">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>40</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleAlpha">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelNumIterations">
     <property name="text">
      <string>Число итераций</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinNumAnts">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="checkTreeMode">
     <property name="text">
      <string>Дерево путей</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AntAlgorithmDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AntAlgorithmDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
